spec/lib/goodcity/organisation_populator_spec.rb

```diff
@@ -44,12 +44,13 @@
       end
 
       it ":update the existing records" do
+        organisation_populator.run
```

Comment:
@trojanh before this change, we were effectively testing that nothing changes when nothing is called! Now we're testing that updates are applied when the populator is run.